packages/prisma/schema.prisma

```diff
@@ -451,6 +454,9 @@ model Booking {
   uid                   String               @unique
   user                  User?                @relation(fields: [userId], references: [id], onDelete: Cascade)
   userId                Int?
+  // User's email at the time of booking
+  /// @zod.email()
+  userPrimaryEmail      String?
```

Comment:
We need to store only user's email at the time of booking so that even when user changes email or connect another calendar we display the email at the time of booking on /bookings/[uid] and use that

Reply:
For attendees we save that info in Attendee table and that is not updated even when any attendee changes email as  that new info is stored in User Table